src/components/MainProductBox.js

```diff
@@ -1,7 +1,13 @@
 import { Typography, Box, Button } from '@mui/material';
 import CircleIcon from '@mui/icons-material/Circle';
 import { PrimaryButton } from './Buttons';
+import { useHistory } from 'react-router-dom';
 const MainProductBox = () => {
+  const history = useHistory();
+
+  const toProductDashboard = () => {
+    history.push('/');
```

Comment:
This will lead to the landing page. Change it what your dashboard route will be.